troposphere/static/js/components/projects/detail/details/ViewDetails.jsx

```diff
@@ -61,8 +63,9 @@ const ViewDetails = React.createClass({
         return group_users_text.join(", ");
     },
     renderLeaders: function(project) {
+        let { GroupStore } = this.props.subscriptions;
```

Comment:
Credit goes to `eslint` on this one for seeing the "stores" reference. But the `--fix` for it likely would have been wrong.